src/platform/graphics/texture.js

```diff
@@ -955,8 +955,13 @@ class Texture {
                     this._levelsUpdated[mipLevel] = true;
                 }
 
-                width = source.width;
-                height = source.height;
+                if (source instanceof HTMLVideoElement) {
```

Comment:
not sure how applicable these days, but in the graphics device we test for presence of HTMLVideoElement as well:
`(typeof HTMLVideoElement !== 'undefined' && texture instanceof HTMLVideoElement)`

Reply:
Oh, that's probably for running in Node? There seem to already be a couple of other `instanceof HTMLVidelElement` tests in the platform _without_ the additional guard so I think we're fine.